optaweb-vehicle-routing-standalone/pom.xml

```diff
@@ -52,10 +53,17 @@
       <artifactId>optaweb-vehicle-routing-frontend</artifactId>
       <type>war</type>
     </dependency>
-    <!-- Used by WebConfig. -->
     <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-web</artifactId>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-undertow</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-h2</artifactId>
     </dependency>
```

Comment:
Shouldn't one of the two be optional? Why have both on the classpath?

Reply:
Yes. I do consider that acceptable, considering that this is a template, and not a production application.